### ISSUE TITLE
Better MC [PLUS] BMP2 Crashing on launch

### ISSUE THREAD

**Issue by Adzeiros** [Autovw/AdvancedNetherite#25]
**Describe the crash / error**
I installed just the Advanced Netherite mod into the Better MC [PLUS] BMP2 modpack (1.18.2 Forge) Launch the game and it crashes before even showing the initial loading screen.

**To Reproduce**
Install the mod into the Better MC [PLUS] BMP2 modpack and launch

**Expected behavior**
The game should run fine as the Game Version is correct,

**Logs**
Please provide a crash log (debug.log or latest.log)! Crashlogs can be found inside Minecraft's ``logs`` folder. Please use a paste website when sharing the log, e.g. [Pastebin](https://pastebin.com) or [Gist](https://gist.github.com).

https://gist.github.com/Adzeiros/e63813e0f38970e899aa355363fa5840

**Technical Information (please complete the following information):**
 ! Do not say 'latest', provide the actual version.
 - Minecraft-version: 1.18.2
 - Forge-version: 40.1.69
 - Mod-version: 1.14.6

**Comment by Autovw** [comment]
The log seems to be cut short and does not say anything useful. However, you are running an outdated version of Forge. Advanced Netherite requires at least ``40.1.84`` to be installed. The current recommended Forge version is even higher. The game should tell you this, but something else is preventing the game from showing the loading screen for some reason. This can't be Advanced Netherite as it's not supposed to do anything at the stage your log ends.

In short: try updating your modpack, or report to the modpack author if this does not work.

**Comment by Adzeiros** [comment]
> The log seems to be cut short and does not say anything useful. However, you are running an outdated version of Forge. Advanced Netherite requires at least `40.1.84` to be installed. The current recommended Forge version is even higher. The game should tell you this, but something else is preventing the game from showing the loading screen for some reason. This can't be Advanced Netherite as it's not supposed to do anything at the stage your log ends.
> 
> In short: try updating your modpack, or report to the modpack author if this does not work.

Couldn't tell you, all I know is it only crashes when I add Advanced Netherite to it. It works fine with the other mods I've added. I'll keep it in mind though and see if I can update the forge version without causing issues to the base Modpack.
